tests/HotkeysProvider.test.tsx

```diff
@@ -220,4 +239,21 @@ test('should return bound hotkeys when defined as a string array', () => {
   })
   expect(result.current.hotkeys[0].keys).toEqual(['a', 'c'])
   expect(result.current.hotkeys[1].keys).toEqual(['b'])
-})
\ No newline at end of file
+})
+
+test('should return descriptions for bound hotkeys', () => {
```

Comment:
The same test appears in Line 181

Reply:
Sorry...meant to move it, and left a copy. I also reverted a bad merge in `useHotkeys.ts` that broke this feature.